src/control/server/storage/scm/ipmctl.go

```diff
@@ -444,6 +476,11 @@ func (cr *cmdRunner) prep(scanRes *storage.ScmScanResponse) (resp *storage.ScmPr
 		return nil, err
 	}
 
+	// Handle unspecified NrNamespacesPerNUMA in request.
+	if req.NrNamespacesPerNUMA == 0 {
+		req.NrNamespacesPerNUMA = 1
```

Comment:
Would probably be better to make this a const, e.g. `DefaultNrNamespacesPerNUMA`.

Reply:
done